tests/nest_tests/third_factor_stdp_synapse_test.py

```diff
@@ -167,8 +161,11 @@ def run_synapse_test(self, neuron_model_name,
                 state = 1
             if t > 2 * sim_time / 6 and state == 1:
                 nest.SetStatus(post_neuron, {"I_dend": 1.})
-            if t > 2 * sim_time / 3. and state == 1:
+            if t > 3 * sim_time / 6. and state == 1:
```

Comment:
Could you maybe comment what the states represent here?

Reply:
It's a sequence of statements to apply an externally injected current (and stop applying it) at given times. I don't think the states can be given meaningful names, hence the use of integers.